compose/gettingstarted.md

```diff
@@ -98,73 +96,142 @@ This Compose file defines two services, `web` and `redis`. The web service:
 * Uses an image that's built from the `Dockerfile` in the current directory.
 * Forwards the exposed port 5000 on the container to port 5000 on the host
   machine. We use the default port for the Flask web server, `5000`.
-* Mounts the project directory on the host to `/code` inside the container,
-  allowing you to modify the code without having to rebuild the image.
 
 The `redis` service uses a public
 [Redis](https://registry.hub.docker.com/_/redis/) image pulled from the Docker
 Hub registry.
 
->**Tip**: If your project is outside of the `Users` directory (`cd ~`), then you
-need to share the drive or location of the Dockerfile and volume you are using.
-If you get runtime errors indicating an application file is not found, a volume
-mount is denied, or a service cannot start, try enabling file or drive sharing.
-Volume mounting requires shared drives for projects that live outside of
-`C:\Users` (Windows) or `/Users` (Mac), and is required for _any_ project on
-Docker for Windows that uses [Linux
-containers](/docker-for-windows/#switch-between-windows-and-linux-containers-beta-feature).
-For more information, see [Shared
-Drives](../docker-for-windows/#shared-drives) on Docker for Windows,
-[File sharing](../docker-for-mac/#file-sharing) on Docker for Mac, and
-the general examples on how to [Manage data in
-containers](../engine/tutorials/dockervolumes.md).
-
 ## Step 4: Build and run your app with Compose
 
-1. From your project directory, start up your application.
+1.  From your project directory, start up your application.
+
+    ```
+    $ docker-compose up
+    Creating network "composetest_default" with the default driver
```

Comment:
@mstanleyjones the extra line doesn't exist on the terminal .. I can add it, but it isn't representative of what the user will see. Leaving for now, we can chat about it tomorrow. Samples in Compose, d4mac, d4win, Machine show the input / output as it actually displays.